### ISSUE TITLE
dubious auto-config of threads per process

### ISSUE THREAD

**Issue by ornicar** [niklasf/fishnet#71]
Using fishnet defaults on my 8-cores machines.

- Expected behavior: 2 engine processes of 4 and 3 threads each
- Observed behavior: 1 engine process of 7 threads.

```
> fishnet ±(master 4467c) ./fishnet.py 
.   _________         .    .
.  (..       \_    ,  |\  /|
.   \       O  \  /|  \ \/ /
.    \______    \/ |   \  /      _____ _     _     _   _      _
.       vvvv\    \ |   /  |     |  ___(_)___| |__ | \ | | ___| |_
.       \^^^^  ==   \_/   |     | |_  | / __| '_ \|  \| |/ _ \ __|
.        `\_   ===    \.  |     |  _| | \__ \ | | | |\  |  __/ |_
.        / /\_   \ /      |     |_|   |_|___/_| |_|_| \_|\___|\__| 1.13.0
.        |/   \_  \|      /
.               \________/      Distributed Stockfish analysis for lichess.org


### Configuration

Engine working directory (default: /home/thib/fishnet): 

[...]

Path or command (will download by default): 

Number of cores to use for engine threads (default 7, max 8): 
Configure advanced options? (default: no) 
Personal fishnet key (append ! to force, required): offline!

Done. Write configuration to /home/thib/fishnet/fishnet.ini now? (default: yes) 
Configuration saved.

### Updating Stockfish ...

Engine target path: /home/thib/fishnet/stockfish-x86_64-bmi2
Looking up stockfish-x86_64-bmi2 ...
Local stockfish-x86_64-bmi2 is newer than release

### Checking configuration ...

EngineDir:        /home/thib/fishnet
StockfishCommand: ./stockfish-x86_64-bmi2
Key:              *******
Cores:            7
Engine processes: 1 (each ~4 threads)
Memory:           256 MB
Endpoint:         https://en.lichess.org/fishnet/
FixedBackoff:     False

### Starting workers ...

><> 1: Started Stockfish 250417 64 BMI2, threads: +++++++ (7), pid: 27600
><> 1: ENGINE: 27598 >> option name Syzygy50MoveRule type check default true
><> 1: ENGINE: 27598 >> option name SyzygyProbeLimit type spin default 6 min 0 max 6
><> 1: ENGINE: 27598 >> uciok
><> 1: DEBUG: Supported variants: horde, racingkings, kingofthehill, chess, crazyhouse, 3check, bughouse, loop, losers, suicide, giveaway, atomic, var
><> 1: ENGINE: 27600 << uci
><> 1: ENGINE: 27600 >> Stockfish 250417 64 BMI2 by T. Romstad, M. Costalba, J. Kiiski, G. Linscott
><> 1: ENGINE: 27600 >> id name Stockfish 250417 64 BMI2
><> 1: ENGINE: 27600 >> id author T. Romstad, M. Costalba, J. Kiiski, G. Linscott
><> 1: ENGINE: 27600 >> 
[...]
><> 1: ENGINE: 27600 >> uciok
><> 1: ENGINE: 27600 << setoption name threads value 7
><> 1: ENGINE: 27600 << setoption name hash value 256
```

**Comment by niklasf** [comment]
released v1.13.1 with this tweak

**Comment by Alexj66** [comment]
Hello, I sent the request for receive the fishnet API key.
But still i didn't receive. How to should i do? 
I want you will let me know. 
Thanks.
![Capture](https://user-images.githubusercontent.com/50274766/57857078-05d54080-77f7-11e9-90ca-34208dbd21f6.PNG)
